package.json

```diff
@@ -14,19 +14,10 @@
     "build": "node build/build.js"
   },
   "dependencies": {
-    "common-component": "git+https://github.com/Rise-Vision/common-component.git#v1.5.1",
+    "common-component": "git+https://github.com/Rise-Vision/common-component.git#1.5.5",
```

Comment:
@rodrigopavezi btw you're missing the `v` in the version numbers for releases as is common practice. I'll fix the releases in `common-component` and this reference before merging since you're away, thanks. 

@pcsandford FYI a note for the future when you're releasing versions 👍